src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -84,7 +84,7 @@ public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDecla
 
 						// The regex here is mainly for the \s as a convenient way to remove whatever whitespace TIDY
 						// has inserted. It's a fringe benefit that we can use the[bi] to deal with both elements in one replace.
```

Comment:
This comment is out of date (at least mentioning only [bi] but now that it replaces about 5 options it's really obsolete altogether).